activities/office-management/index.md

```diff
@@ -0,0 +1,9 @@
+---
+type: Index
+---
+
+# Office management
+
+Our office houses the beating heart of the organization. We're all responsible for keeping it working and enjoyable.
+
+* [Using our space guide](using-space.md): What roles spaces have and what their code of conduct is
```

Comment:
Perhaps should contain a period at the end, maybe not end the sentence with the verb. Does not block merge.